### PR TITLE
Copied the files from my last project for peer to peer code review se…

### PR DESCRIPTION
**Pull request by safar1212**
My partner can examine the code and present his valuable recommandations.

### CODE REVIEW

**Comment by safar1212** [comment]
@Josphat205

File: src/listItem.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+let toDos = JSON.parse(localStorage.getItem('todo-list')) || [];
+
+function listItems() {
+  toDos = JSON.parse(localStorage.getItem('todo-list')) || [];
```
Comment (Josphat205) [inline src/listItem.js:4]: - [ ] Please make your codes DRY (don't repeat yourself) toDos is Defined in line 1, its a Global variable you can use it in the function,  don't repeat in line 4, go on and use toDos variable.

**Review by Josphat205 — CHANGES_REQUESTED** [review]
### Hi @safar1212 
68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f4b7942657366386b4e5436733838534567622f67697068792e676966


## CHANGES REQUIRED ♻️ 
## HIGHLIGHTS
- [x] Professional Readme file
- [x] No linters File
- [x] JavaScript best practices
- [x] HTML and CSS best practices
### Recommendation
Hi, Safar Ali, you are really doing a nice job congratulations, but there are a few changes you have to make, check the highlighted codes below.

File: src/listItem.js (hunk 1 above)
Comment (safar1212) [reply src/listItem.js:4]: I will try to change it. But it causes problems when I try to delete all the completed tasks.

>

**Pull request merged**